### PR TITLE
hotfix

### PR DESCRIPTION
**Pull request by AlecRosenbaum**

### CODE REVIEW

File: src/main/java/trackmodel/TrackModel.java [hunk 1]
```diff
@@ -1548,7 +1548,11 @@ public int getTrainPassengers(int trainId) {
             // load waiting passengers
             Double passengers = this.waitingPassengers.get(this.getTrainBlock(trainId));
             if (passengers != null) {
-                int loaded = this.random.nextInt(Math.min((int) passengers.doubleValue(), train.getMaxPassengers()));
+                int bound = Math.min((int) passengers.doubleValue(), train.getMaxPassengers());
+                int loaded = 0;
+                if (bound > 0) {
+                    loaded = this.random.nextInt();
```
Comment (iggy14750) [inline src/main/java/trackmodel/TrackModel.java]: Need to add `bound` as a parameter to `nextInt()`. With no params, [`nextInt()`](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#nextInt--) returns a uniformly distributed value from 0 to 2<sup>32</sup>.

**Review by iggy14750 — CHANGES_REQUESTED** [review]
Need to add `bound` as a parameter to `nextInt()`. With no params, [`nextInt()`](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#nextInt--) returns a uniformly distributed value from 0 to 2<sup>32</sup>.

**Comment by AlecRosenbaum** [comment]
@iggy14750 I don't know why so many commits are showing up, but I just added bound to the nextInt call

**Review by wyakpara — APPROVED** [review]
lgtm

**Review by iggy14750 — APPROVED** [review]

**Pull request merged**